packages/helpwave_service/pubspec.yaml

```diff
@@ -12,6 +12,11 @@ dependencies:
     sdk: flutter
   shared_preferences: ^2.0.15
 
+  impulse:
```

Comment:
We really shouldn't import an app into the packages, just move the entire user service to the impulse app and we will discuss a appropriate abstraction in a different issue